app/src/main/res/values/strings.xml

```diff
@@ -11,6 +11,13 @@
     <string name="sample_coords" translatable="false">55.332244 34.442211</string>
     <string name="sample_message" translatable="false">hey I found the cache, it is over here next to the big tiger. I\'m kinda scared.</string>
 
+    <!-- Main Tabs -->
+    <string name="main_tab_lbl_messages">Messages</string>
+    <string name="main_tab_lbl_users">Users</string>
+    <string name="main_tab_lbl_map">Map</string>
+    <string name="main_tab_lbl_channel">Channel</string>
+    <string name="main_tab_lbl_settings">Settings</string>
```

Comment:
can you add `translatable="false"` into these? I don't want to send them for translation just yet.

Reply:
the plugin can be a pain, so just to be safe. but this is an exception, these names are ancient and you'll notice some variation over time.